src/wasm-stack.h

```diff
@@ -954,6 +958,47 @@ void StackWriter<Mode, Parent>::visitSIMDShift(SIMDShift* curr) {
   }
 }
 
+template<StackWriterMode Mode, typename Parent>
+void StackWriter<Mode, Parent>::visitMemoryInit(MemoryInit* curr) {
+  visitChild(curr->dest);
+  visitChild(curr->offset);
+  visitChild(curr->size);
+  if (justAddToStack(curr)) return;
+  o << int8_t(BinaryConsts::TruncSatPrefix);
+  o << U32LEB(BinaryConsts::MemoryInit);
```

Comment:
Do the bulk memory operatoins use the trunc_sat prefix?

Reply:
They do. This prefix should probably be renamed to `MiscPrefix`, as it is called in the spec proposals.

Reply:
I see, thanks. Yeah, let's rename to that then, looks odd otherwise.